bot/telegram/telegram_bot.go

```diff
@@ -44,6 +49,40 @@ func (b *Bot) Serve() error {
 
 	updates := bot.GetUpdatesChan(u)
 
+	location, err := time.LoadLocation("Asia/Ho_Chi_Minh")
```

Comment:
Should plit into different function, add "Asia/Ho_Chi_Minh" inside [constant package](https://github.com/hvn2706/jsin/blob/67c210da41c22fa2ade3a028a9915b7ad924780c/pkg/constants/constant.go)

Reply:
Good catch. Thanks